doc/manual/src/package-management/terminology.md

```diff
@@ -0,0 +1,28 @@
+# Terminology
+
+A *local store* exists on the local filesystem of the machine where
+Nix is invoked.  The `/nix/store` directory is one example of a
+local store.  You can use other local stores by passing the
+`--store` flag to `nix`.
+
+A *remote store* is a store which exists anywhere other than the
+local filesystem.  One example is the `/nix/store` directory on
+another machine, accessed via `ssh` or served by the `nix-serve`
+Perl script.
```

Comment:
```suggestion
From   the  perspective   of   the  location   where  Nix   is
invoked<sup><b>1</b></sup>, the  Nix store can be  referred to
as a "_local_" or a "_remote_" one:

<sup>\[1]: Where "invoking Nix" means  an executing a Nix core
action/operation on  a Nix store.  For example, using  any CLI
commands from the `NixOS/nix` implementation.</sup>

+ A *local  store* exists  on the local  filesystem of
  the machine where Nix is  invoked. You can use other
  local stores  by passing  the `--store` flag  to the
  `nix` command.

+ A  *remote store*  exists  anywhere  other than  the
  local  filesystem. One  example is  the `/nix/store`
  directory on another machine,  accessed via `ssh` or
  served by the `nix-serve` Perl script.
```

I also removed the sentence "_The `/nix/store` directory is one example of a local store._" because the term "_Nix store_" should also be in the glossary. (The only place where it is defined is the [Introduction](https://nixos.org/manual/nix/unstable/introduction.html); planning to work on a PR about that.)

The footnote is mostly a placeholder that it should be explained what we mean by "invoking Nix".

Reply:
Since this stuff belongs in the glossary anyways, I propose to implement your suggestion above, and then move the entire chunk into the glossary as a starting point for the definition of "Nix store".

Obviously it isn't a complete entry since it still doesn't say what a "Nix store" is, but when that definition is added the two sub-cases will already be in the right place.

Reply:
Sorry, meant to but subsequently forgot to open a placeholder issue for a new glossary entry for "Nix store". Here it is: #6875